BesterLive/AppDelegate.swift

```diff
@@ -14,23 +14,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
 
 
-  func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) ->Bool{
+  func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject:AnyObject]?) -> Bool {
     // Override point for customization after application launch.
     print("helllllo")
     return true
   }
 
-  func applicationWillResignActive(application:UIApplication){
+  func applicationWillResignActive(application: UIApplication) {
     // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
```

Comment:
Reporter: CHECKSTYLE
Rule: ?
Severity: ERROR
File: /Users/framgia/.jenkins/jobs/BesterLive/workspace/BesterLive/AppDelegate.swift L24
 

Line should be 100 characters or less: currently 281 characters

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>